torch/autograd/variable.py

```diff
@@ -259,6 +268,15 @@ def btrifact(self, info=None, pivot=True):
         else:
             return super(Variable, self).btrifact(pivot=pivot)
 
+    @torch.onnx.symbolic_override(_size_as_tensor)
+    def size_as_tensor(self):
```

Comment:
You've just added two functions to the variable API. Prefix them with underscores